vendor.conf

```diff
@@ -58,7 +58,7 @@ github.com/cilium/ebpf                              1c8d4c9ef7759622653a1d319284
 
 # cri dependencies
 github.com/davecgh/go-spew                          v1.1.1
-github.com/docker/docker                            4634ce647cf2ce2c6031129ccd109e557244986f
+github.com/docker/docker                            9c15e82f19b0ad3c5fe8617a8ec2dddc6639f40a
```

Comment:
Looks like the updated docker vendor does not bring in any changes; was there a specific need to update?

Reply:
the SHA was older than the one in k/k, so was trying to update to newer one. am ok to leave this as-is. 

i had to jump between versions of `containerd/console` `x/sys` and `docker/docker` to find the right combination that works and i have to do the same thing in cadvisor and k/k, i thought i would file this here first :)

Reply:
Looks like there's only two packages used from docker; perhaps worth evaluating if those could be separated 🤔 (the "contrib" files are vendored because they contain C files, which `vndr` always vendors)

<img width="249" alt="Screenshot 2020-10-13 at 23 26 46" src="https://user-images.githubusercontent.com/1804568/95918410-0ab37080-0dac-11eb-9860-79f79d0c6750.png">